src/TsTypes.ts

```diff
@@ -1,5 +1,9 @@
 import { camelCase, upperFirst } from 'lodash'
 
+const multiLineCommentStart =  '/** '
+const multiLineCommentIndent = '    '
```

Comment:
Bummer- I was trying to find a comment syntax that VSCode will use to inform its intellisense, but it doesn't look like comments for interfaces are supported.